### ISSUE TITLE
R4 - TYPE_NAME_RAW constant missing.

### ISSUE THREAD

**Issue by iBotPeaches** [dcarbone/php-fhir#85]
I generated from the latest master a new copy of R4 and ran the test suite, skipping Integration tests.

```

1) HL7\FHIR\R4\PHPFHIRTests\Unit\FHIRRawTest::testCanConstructTypeNoArgs
Error: Undefined constant HL7\FHIR\R4\PHPFHIRConstants::TYPE_NAME_RAW

/Volumes/Nexus/Github/php-fhir-generated-r4/src/HL7/FHIR/R4/FHIRRaw.php:77

ERRORS!
Tests: 2118, Assertions: 9952, Errors: 1.
```

![Screenshot 2023-01-12 at 11 44 30 AM](https://user-images.githubusercontent.com/611784/212128107-ac33548b-44ab-4fcc-99f7-b258e381a248.png)

It does appear that constant is now missing. Unsure at the moment if this was parser changes or some change to R4.

**Comment by dcarbone** [comment]
Could you try re-running the generator, this time not skipping test generation?  `master` has quite a few updates to it, including in the tests.

**Comment by iBotPeaches** [comment]
Yeah sorry for confusion. I did generate it all, then literally just took Intergration tests out of phpunit.xml, since those were failing with some odd double wrapped div errors.

Eirher way, I'll clear out everything and run it again.
